ncolony/process_events.py

```diff
@@ -44,6 +45,10 @@ def add(self, name, contents):
                           if key in VALID_KEYS}
         parsedContents['name'] = name
         parsedContents['env'] = parsedContents.get('env', {})
+        for key, value in list(parsedContents['env'].items()):
+            if value is not None:
```

Comment:
can't tell from unit tests -- this would be in the json file as null?

Reply:
Yes. (And currently isn't supported in the command line)

Reply:
Using `None`/`null` to mean "inherit" seems confusing.  To me, it signals *not* inheriting the variable, where the default is inheritance.

It would be better to use an explicit sentinel, e.g. `{"inherit": true}`, or separate parameters for explicitly provided environment variables and inherited variables, e.g.:

```json
{
  "env":
  {
    "values": {"SOMETHING": "/a/path"},
    "inherited": ["PATH"]
  }
}
```

I prefer the second approach because it's more explicit.  Inheriting environment variables is a great way to get bugs, so I think the verbosity is a fair price to pay.

Reply:
I did something along the lines of what @markrwilliams suggested, but without breaking backwards compat -- I added a separate top-level key, "env_inherit", which maps to a list of variables to inherit.